src/sources/aws_ecs_metrics/mod.rs

```diff
@@ -186,16 +186,10 @@ async fn aws_ecs_metrics(
             .expect("error creating request");
         let uri = request.uri().clone();
 
-        let start = Instant::now();
```

Comment:
Hmm this source is actually using `hyperClient` instead of `crate::http::HttpClient`, meaning that it doesn't include our default instrumentation. Can we swap out the client here?

Reply:
Ah, good catch. I'll update.